rules/library.bzl

```diff
@@ -1005,7 +1005,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             private_dep_names.append(swift_doublequote_hmap_name)
             _append_headermap_copts(swift_doublequote_hmap_name, "-iquote", additional_objc_copts, additional_swift_copts, additional_cc_copts)
 
-            # Add generated swift header to header maps for double quote imports
+            # Add generated swift header to header maps for angle bracket imports
```

Comment:
AFAIK this is `double quote` *and* `angle bracket` - not sure if there is some quirk that required them both!